DOC-2729 micro app templates (#8)
lumapps-marketplace/lumapps-extension-code-sample: micro-app/Workday leave request/README.md

**Prerequisites:**

- Read the [Micro-app step-by-step guide](https://developer.lumapps.com/portal/documentation/micro-app-steps/micro_app_step_by_step.md) documentation.
    
- Create a connector for the service provider you will use according to [Extension connectors](https://docs.lumapps.com/docs/admin-l43084339674928007extensions).

The workflow contains the following elements:

1. Get the worker ID of the user.
2. An API Call to get the remaining days off of the user.
3. An Adaptive Card to display a leave request form.
4. An API Call to send the leave request data to Workday.
5. A router and filters to create different workflows:
    - 1 branch for when the leave request is successful
    - 1 branch for when the leave request fails
6. An Adaptive Card for each branch, to display either an error or a success message.

> **Prerequisite:** Create a new micro-app according to [Declare your micro-app extension](https://developer.lumapps.com/portal/documentation/micro-app-steps/declare-your-microapp-extension.md).

Create the elements of the workflow according to the steps in this page and the Developer Portal documentation.

## Step 1: Get the worker ID of the user

> In this example, users have the same value for their LumApps employeeId as their Workday workerId. So we use a LumApps API to retrieve this ID.

Reviewer: ```suggestion
In this example, users have the same value for their LumApps employeeId as their Workday workerId. Therefore, you use a LumApps API to retrieve this ID.
```
